tidb-cloud/terraform-use-import-resource.md

```diff
@@ -0,0 +1,235 @@
+               result     = "SUCCESS"
+               table_name = "`test`.`import_test`"
+           },
+       ]
+       cluster_id                    = "1379661944641274168"
+       completed_percent             = 100
+       completed_tables              = 1
+       created_at                    = "2023-02-06T05:39:46.000Z"
+       csv_format                    = {
+           separator = ";"
+       }
+       data_format                   = "CSV"
+       elapsed_time_seconds          = 48
+       file_name                     = "./t.csv"
+       id                            = "781074"
+       new_file_name                 = "2023-02-06T05:39:42Z-t.csv"
+       pending_tables                = 0
+       post_import_completed_percent = 100
+       processed_source_data_size    = "31"
+       project_id                    = "1372813089191151295"
+       status                        = "IMPORTING"
+       target_table                  = {
+           database = "test"
+           table  = "import_test"
+       }
+       total_files                   = 0
+       total_size                    = "31"
+       total_tables_count            = 1
+       type                          = "LOCAL"
+   }
+   ```
+
+5. Use `terraform refresh` to update the status after several minutes:
+
+   ```
+   $ terraform refresh && terraform state show tidbcloud_import.example_local
+   tidbcloud_import.example_local: Refreshing state... [id=781074]
+   # tidbcloud_import.example_local:
+   resource "tidbcloud_import" "example_local" {
+       all_completed_tables          = [
+           {
+               message    = ""
+               result     = "SUCCESS"
+               table_name = "`test`.`import_test`"
+           },
+       ]
+       cluster_id                    = "1379661944641274168"
+       completed_percent             = 100
+       completed_tables              = 1
+       created_at                    = "2023-02-06T05:39:46.000Z"
+       csv_format                    = {
+           separator = ";"
+       }
+       data_format                   = "CSV"
+       elapsed_time_seconds          = 49
+       file_name                     = "./t.csv"
+       id                            = "781074"
+       new_file_name                 = "2023-02-06T05:39:42Z-t.csv"
+       pending_tables                = 0
+       post_import_completed_percent = 100
+       processed_source_data_size    = "31"
+       project_id                    = "1372813089191151295"
+       status                        = "COMPLETED"
+       target_table                  = {
+           database = "test"
+           table  = "import_test"
+       }
+       total_files                   = 0
+       total_size                    = "31"
+       total_tables_count            = 1
+       type                          = "LOCAL"
+   }
+   ```
+
+   When the status turns to `COMPLETED`, it indicates that the import task is finished.
+
+6. Check the import data with MySQL CLI:
```

Comment:
Do we need to mention "MySQL CLI" or MySQL clients in the prerequisites?

Reply:
I think we needn't. It is not a necessary step for import, just check the results for import